### ISSUE TITLE
Functional.copy() does not support models with dropout layer

### ISSUE THREAD

**Issue by rychuelektryk** [JetBrains/KotlinDL#329]
Exception in thread "main" java.lang.IllegalStateException: dropout_3 with type Dropout is not supported yet!
	at org.jetbrains.kotlinx.dl.api.inference.keras.ModelSaverKt.convertToKerasLayer(ModelSaver.kt:111)
	at org.jetbrains.kotlinx.dl.api.inference.keras.ModelSaverKt.serializeModel(ModelSaver.kt:46)
	at org.jetbrains.kotlinx.dl.api.core.Functional.copy(Functional.kt:327)

It looks like method convertToKerasLayer does not handle Dropout layer yet

**Comment by zaleslaw** [comment]
Thanks for the issue!
